### ISSUE TITLE
How to use BiDynamicRNNLayer for Text classification?Do not support return_last at the moment ?

### ISSUE THREAD

**Issue by chaiyixuan** [tensorlayer/tensorlayer#392]
self.x = tf.placeholder("float", [None, None, alphabet_size], name="inputs")
        self.y = tf.placeholder(tf.int64, [None, ], name="labels")
        self.dropout_keep_prob = tf.placeholder(tf.float32, name="dropout_keep_prob")
        #n_hidden = 64 # hidden layer num of features
        self.network = tl.layers.InputLayer(self.x, name='input_layer')
        self.network = tl.layers.BiDynamicRNNLayer(self.network,
            cell_fn         = tf.contrib.rnn.BasicLSTMCell,
            n_hidden        = n_hidden,
            dropout         = dropout_keep_prob,
            sequence_length = tl.layers.retrieve_seq_length_op(self.x),
            return_seq_2d   = True,
            return_last     = True,
            n_layer         = 3,
            name            = 'dynamic_rnn')
        self.network = tl.layers.DenseLayer(self.network, n_units=2,
                                       act=tf.identity, name="output")
        self.network.outputs_op = tf.argmax(tf.nn.softmax(self.network.outputs), 1)
        self.loss      = tl.cost.cross_entropy(self.network.outputs, self.y, 'xentropy')
Raise Exception :Do not support return_last at the moment 

why?

**Comment by PayneJoe** [comment]
Any improvement about this?

**Comment by zsdonghao** [comment]
feel free to reopen if you want to discuss.

we just release 2.0.0 version
